packages/plugin-repo-cli/commands/analyzePlugin.ts

```
@@ -0,0 +1,41 @@
+            'docker', 'run', '--rm',
+            '-v', `${pluginPath}:/src`,
+            'returntocorp/semgrep',
+            'semgrep',
+            '--config', 'p/security-audit',
+            '--config', 'p/typescript',
+            '--error', 
+            '/src'
+        ]);
+        console.info('No critical security issues found.');
+        console.log(output);
+    } catch (error) {
```

ESLint

[error] Expected indentation of 1 tab but found 4 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 4 spaces.

(indent)